awareness.js

```diff
@@ -156,6 +156,54 @@ export class Awareness extends Observable {
   }
 }
 
+/**
+ * A state encoder that serializes state into JSON.
+ *
+ * This is the default strategy used in this package.
+ */
+export class DefaultAwarenessStateEncoder {
+  /**
+   * Encode an awareness state entry
+   * @param {encoding.Encoder} encoder
+   * @param {any} update
+   */
+  encodeState (encoder, update) {
```

Comment:
These should be `static` methods. Then there is no need to create an instance of `*AwarenessStateEncoder`

Reply:
Sure, can do.